### ISSUE TITLE
Will it create new directory if directory does not exist at s3 bucket.

### ISSUE THREAD

**Issue by veeresh-sd** [jakejarvis/s3-sync-action#28]
I have specified DEST_DIR in the environment variables.
DEST_DIR is something like this: 'test/abc/xyz'
Assume the xyz directory does not exist at s3 bucket, will it create a new directory with the same name xyz. If not, how can I make it to create a directory and then push it?
I found an issue with the below error message

 ```
Could not connect to the endpoint URL: 
"https://***.s3.***.amazonaws.com/?list-type=2&prefix=test%abc%xyz%2F&encoding-type=url"
  Post Run actions/checkout@v2
```

**Comment by mukunku** [comment]
Yes it will because S3 doesn't have the concept of directories. It's just another Key that happens to have slashes in it.
